secret_number.js

```diff
@@ -1,4 +1,40 @@
+//var secretNumber = function(){
 'use strict';
 module.exports = function() {
-  
+
+
+var sameRandomNumber = 0;
+var genRandomNumber = Math.random();
+
+  return function(){
+  sameRandomNumber = genRandomNumber*1000001;
```

Comment:
So at this point, genRandomNumber is Math.random() * 1000001 which at the highest is 1000000.99999 which is slightly above the limit of 1000000. Can you think of any other Math method that will allow you to hit 1000000 exactly?